prime-router/metadata/schemas/covid-19.schema

```diff
@@ -150,8 +150,8 @@ elements:
 
   - name: filler_order_id
     type: ID
-    hl7Field: ORC-3-2
-    hl7OutputFields: [ ORC-3-2, SPM-2-2, OBR-3-2 ]
+    hl7Field: ORC-3-1
+    hl7OutputFields: [ ORC-3-1, SPM-2-2, OBR-3-1 ]
 
```

Comment:
I've moved the fields around slightly to make them pass Guam's requirements, and to make sure that OBR-3-1 and ORC-3-1 are filled in and match.